npm run init on clean database fails with "your account is disabled"

npm run init no longer works on the head of devel for me, log below. Note that the midas_user table has no users in it, I'm not sure if that's how it used to be. 

Steps to repro: run my [reset script](https://gist.github.com/sefk/b044a5dfeda3e0a9fdfd) or do the steps yourself:

1. drop db, redo grants
2. npm run migrate
3. npm run init

Error log:

```
sef@speck (devel) midas> git show
commit cffcdd94fcc864d70dbc184f7be4054d4c918f97
Merge: dbebfff 276ee9b
Author: Dave Cole <dhcole@users.noreply.github.com>
Date:   Thu May 14 11:45:00 2015 -0400

    Merge pull request #774 from 18F/tests

    Browser tests

sef@speck (devel) midas> midas-reset.sh
This will nuke and reload your midas db, are you sure [y/N] y
+ psql -c 'drop database midas;'
'DROP DATABASE
+ psql -c 'create database midas;'
CREATE DATABASE
+ psql -c 'GRANT ALL PRIVILEGES ON DATABASE midas to midas;'
GRANT
+ psql -c 'ALTER SCHEMA public OWNER TO midas;'
ALTER SCHEMA
+ npm run migrate

> midas@0.3.4 migrate /Users/sef/src/18F/midas
> ./tools/postgres/init.sh

Loading current schema.
SET
SET
SET
SET
SET
SET
CREATE EXTENSION
psql:/Users/sef/src/18F/midas/tools/postgres/schema/current.sql:23: ERROR:  must be owner of extension plpgsql
SET
SET
SET
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE TABLE
ALTER TABLE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
CREATE TABLE
ALTER TABLE
CREATE SEQUENCE
ALTER TABLE
ALTER SEQUENCE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
psql:/Users/sef/src/18F/midas/tools/postgres/schema/current.sql:1474: WARNING:  no privileges could be revoked for "public"
REVOKE
psql:/Users/sef/src/18F/midas/tools/postgres/schema/current.sql:1475: WARNING:  no privileges could be revoked for "public"
REVOKE
psql:/Users/sef/src/18F/midas/tools/postgres/schema/current.sql:1476: WARNING:  no privileges were granted for "public"
GRANT
psql:/Users/sef/src/18F/midas/tools/postgres/schema/current.sql:1477: WARNING:  no privileges were granted for "public"
GRANT
INSERT 0 1
+ npm run init

> midas@0.3.4 init /Users/sef/src/18F/midas
> mocha test/test.upstart.js --development --recursive test/init



lifting sails: env=development
Loading...  /Users/sef/src/18F/midas/config/application.js
Loading...  /Users/sef/src/18F/midas/config/assets.js
Loading...  /Users/sef/src/18F/midas/config/auth.js
Loading...  /Users/sef/src/18F/midas/config/settings/auth.js
Loading...  /Users/sef/src/18F/midas/config/bootstrap.js
Loading...  /Users/sef/src/18F/midas/config/connections.js
Loading...  /Users/sef/src/18F/midas/config/controllers.js
Loading...  /Users/sef/src/18F/midas/config/csrf.js
Loading...  /Users/sef/src/18F/midas/config/email.js
Loading...  /Users/sef/src/18F/midas/config/globals.js
Loading...  /Users/sef/src/18F/midas/config/http.js
Loading...  /Users/sef/src/18F/midas/config/io.js
Loading...  /Users/sef/src/18F/midas/config/local.js
Loading...  /Users/sef/src/18F/midas/config/log.js
Loading...  /Users/sef/src/18F/midas/config/models.js
Loading...  /Users/sef/src/18F/midas/config/policies.js
Loading...  /Users/sef/src/18F/midas/config/routes.js
Loading...  /Users/sef/src/18F/midas/config/session.js
Loading...  /Users/sef/src/18F/midas/config/env/development.js
Loading...  /Users/sef/src/18F/midas/config/settings/analytics.js
Loading...  /Users/sef/src/18F/midas/config/settings/auth.js
Loading...  /Users/sef/src/18F/midas/config/settings/emailTemplates.js
Loading...  /Users/sef/src/18F/midas/config/settings/notifications.js
Loading...  /Users/sef/src/18F/midas/config/settings/sources.js
Loading...  /Users/sef/src/18F/midas/config/settings/tags.js
Loading...  /Users/sef/src/18F/midas/config/sockets.js
Loading...  /Users/sef/src/18F/midas/config/version.js
Loading...  /Users/sef/src/18F/midas/config/views.js
Loading...  /Users/sef/src/18F/midas/config/local.js
Loading...  /Users/sef/src/18F/midas/config/env/development.js
  init:
    1) tags


Lowering sails

  0 passing (2s)
  1 failing

  1) init: tags:
     Error: done() invoked with non-Error: Error: Register unsuccessful. {
  "error": null,
  "message": "Your account is disabled."
}
      at /Users/sef/src/18F/midas/node_modules/mocha/lib/runnable.js:239:25
      at /Users/sef/src/18F/midas/test/init/init/init.test.js:34:23
      at Request._callback (/Users/sef/src/18F/midas/test/demo/data/utils.js:39:22)
      at Request.self.callback (/Users/sef/src/18F/midas/node_modules/request/request.js:360:22)
      at Request.emit (events.js:98:17)
      at Request.<anonymous> (/Users/sef/src/18F/midas/node_modules/request/request.js:1202:14)
      at Request.emit (events.js:117:20)
      at IncomingMessage.<anonymous> (/Users/sef/src/18F/midas/node_modules/request/request.js:1150:12)
      at IncomingMessage.emit (events.js:117:20)
      at _stream_readable.js:944:16
      at process._tickDomainCallback (node.js:492:13)




npm ERR! midas@0.3.4 init: `mocha test/test.upstart.js --development --recursive test/init`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the midas@0.3.4 init script.
npm ERR! This is most likely a problem with the midas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     mocha test/test.upstart.js --development --recursive test/init
npm ERR! You can get their info via:
npm ERR!     npm owner ls midas
npm ERR! There is likely additional logging output above.
npm ERR! System Darwin 14.3.0
npm ERR! command "/Users/sef/.nvm/v0.10.38/bin/node" "/Users/sef/.nvm/v0.10.38/bin/npm" "run" "init"
npm ERR! cwd /Users/sef/src/18F/midas
npm ERR! node -v v0.10.38
npm ERR! npm -v 1.4.28
npm ERR! code ELIFECYCLE
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/sef/src/18F/midas/npm-debug.log
npm ERR! not ok code 0
sef@speck (devel) midas> psql midas -c "select * from midas_user;"
 username | name | title | bio | photoId | photoUrl | isAdmin | disabled | passwordAttempts | id | createdAt | updatedAt | deletedAt
----------+------+-------+-----+---------+----------+---------+----------+------------------+----+-----------+-----------+-----------
(0 rows)
```

--- a/config/env/development.js
+++ b/config/env/development.js
@@ -6,7 +6,5 @@ console.log('Loading... ', __filename);
 //         ( it undeletes things that have been soft deleted on sails lift )
 
 module.exports.models = {
-  migrate: 'alter',
-  'adapter': 'disk'
-
-}
+  migrate: 'safe'
+};

